README.md

```diff
@@ -106,7 +106,7 @@ cat file1 file2 > newcombinedfile
 ### b. `chmod`
 Lets you change the read, write, and execute permissions on your files.  
 ```bash
-chmod -options filename
+chmod +x script
 ```
```

Comment:
I was thinking about that; the page generally seems to just give examples without too much explanation which I think is a good approach. This is not meant to be an exhaustive reference, just a showcase of the commonly used commands.